### DISCUSSION TITLE
Basic Setup for NEWBEE

### DISCUSSION THREAD

**Discussion by BenRiveraJr** [thecowan/wallpanel-android#299] [Q&A] [answered]
I have a Ring alarm and some automation devices and I would like to setup an old tablet to be able to display a Clock, weather, Garage Open/Close status, Alarm status and Light switches.  There is nothing in the current quick start on how to do this.  The screenshots shows something like this but it is not being display by default so how is that all configured?

**Comment by thanksmister** [comment, marked as answer]
The application is to display home automation platform dashboards or any web url.   If you want to have all those items displayed on web page or dashboard page, you need to install your own home automation platform like Home Assistant, Node-Red, or similar.  This app is not a home automation platform, it only has extra services to control remotely and to send data to your home automation platform.

**Reply by BenRiveraJr** [reply]
I thought this was an app like SmartThings that could see things in my home and create actions, I guess I misunderstood what it is attempting to do.

**Reply by thanksmister** [reply]
That explains the confusion.  This application supplements your home automation platforms.  This application is to display those pages with motion detection, screensavers, and the ability to get data from the device, as well as communicate with the device using MQTT which is usually included into the platform.  You might want to look at integrating SmartThings into Home Assistant and use Home Assistant to achieve what you want.
